forslag.md

```diff
@@ -1,68 +1,60 @@
+Koden bag offentlige IT-systemer skal med andre ord være tilgængelig for såvel borger som stat. Det vil sikre billigere systemer, højere kvalitet og økonomisk vækst.***
 
 ## Introduktion
 Offentlige IT-projekter udgør en væsentlig del af den infrastruktur, der får Danmark til at fungere.
-Når IT-systemer ikke fungerer, er det i sidste ende borgerne, der betaler &ndash; gennem skattebilletten, eller ved at spilde tid på systemer, der ikke fungerer.
+Når IT-systemer ikke fungerer, er det i sidste ende borgerne, der betaler &ndash; gennem skatten, eller ved at spilde tid på systemer, der ikke fungerer.
 Den offentlige sektor har gang på gang udviklet projekter, der er gået over budget, eller som har slået fuldstændig fejl. I værste fald er de aldrig  blevet taget i brug <sub>[citation needed]</sub>
 Mange af disse problemer kunne være undgået<sup><a name="itu">1</a></sup>.
 
 Med dette forslag sikrer vi åbenhed og rimelighed i vores livsvigtige digitale infrastruktur.
-Lige nu er store dele af vores digitale liv ejet af få private virksomheder. Derfor er vi som land dybt afhængige af dem &ndash; og det er dyrt og inffektivt!
+Lige nu er store dele af vores digitale liv ejet af få private virksomheder. Derfor er vi som land dybt afhængige af dem &ndash; og det er dyrt og ineffektivt!
 Lad os sammen skabe en fremtid hvor Danmark går forrest med billig, moderne og innovativ IT.
 
 ### Offentlig IT er billig IT
 Når staten køber et IT system i dag køber de katten i sækken.
 Staten ejer ikke engang systemet.
 Det gør den virksomhed, der har bygget det.
-Tænk hvis det også gjaldt for din bil: så ville producenten svejse motorhjelmen fast så snart den blev solgt.
-Selv ikke din mekaniker måtte få lov til at kigge i den.
-Så hvis bilen gik i stykker, ville du være tvunget til at køre den ødelagte bil hjem Asien, hvor den blev bygget.
+Tænk hvis det også gjaldt for din bil: Så ville producenten gøre teknikken i bilen utilgængelig for selv din mekaniker, og du ville derfor være helt afgængig af producenten til at reparere bilen igen.
 
 Desværre er det præcis det, der sker med vores offentlige IT.
-Alene i årene 2000-2014 har staten Danmark kastet 2,902 milliarder kroner væk på skrottet IT.<sup><a name="dr8">2</a></sup>
-Det svarer til 585 fuldtidsansatte sygeplejersker i de 14 år.
+Alene i årene 2000-2014 har staten Danmark brugt 2902 millioner kroner på skrottet IT.<sup><a name="dr8">2</a></sup>
+Det svarer til 585 fuldtidsansatte sygeplejersker i 14 år.
 
 Hvis systemerne havde været ejet af staten kunne vi &ndash; ligesom med en ødelagt bilmotor &ndash; redde og genbruge store dele af dem, selv efter de var brudt sammen.
 
-Det er en langt bedre løsning på en bedre digital hverdag i Danmark, fordi det vil gøre os til herre over eget hus. Vi skal kræve
+Det er en langt bedre løsning på en bedre digital hverdag i Danmark, og vi skal derfor kræve
```

Comment:
Kan bedre lide den eksisterende formulering med "herre over eget hus"...

Reply:
Jeg synes "herre over eget hus" er for vagt. Det er uklart præcis hvad der ligger i det. Er det nødvendigvis altid godt at være herre over eget hus?